_posts/2017-05-30-database-encryption.markdown

```diff
@@ -0,0 +1,172 @@
+---
+layout: post
+title: "Database Encryption"
```

Comment:
This is a very broad. Maybe make it more narrow and interesting, like "encrypting Active Record fields without downtime."